components/04-views/product-view/_product-view.scss

```diff
@@ -260,13 +262,19 @@ $product-view__wrapper-margin\@medium                   : 0 0 40px 0 !default;
 
     &__options {
         width: 100%;
-    }
 
-    .swatch {
-        margin-bottom: $product-view__swatch-margin-bottom;
-        &__option {
-            min-width: $product-view__swatch-option-spacing;
-            min-height: $product-view__swatch-option-spacing;
+        .swatch {
+            margin-bottom: $product-view__swatch-margin-bottom;
+            &__option {
+                min-width: $product-view__swatch-option-spacing;
+                min-height: $product-view__swatch-option-spacing;
```

Comment:
interactives e,ements should be at least 48 x 48px of clickable area

Reply:
Done